src/Directory.Build.props

```diff
@@ -26,6 +25,8 @@
     <PackageProjectUrl>$(RepositoryUrl)</PackageProjectUrl>
     <PackageReleaseNotes>$(RepositoryUrl)/releases/</PackageReleaseNotes>
     <PackageTags>Microsoft Durable Task Orchestration Workflow Activity Reliable DTFx</PackageTags>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
```

Comment:
I've never dealt with snupkg's. How do they get published?

Reply:
The same as a nupkg - `nuget push *.snupkg`